### DISCUSSION TITLE
invalid time error

### DISCUSSION THREAD

**Discussion by robbhar** [tomasrudh/analogclock#23] [Q&A]
I am getting an invalid time error when using this:

type: custom:analog-clock
hide_SecondHand: true
color_HourHand: '#326ba8'
color_MinuteHand: '#3273a8'
color_DigitalTime: '#CCCCC'
color_FaceDigits: '#a83832'
hide_minorticks: true
show_timezone: true
timezone: America/Denver
timezonedisplayname: Mountain Time
dateformat: MMM Do YYYY
timeformat: hh mm

The resource for moment.js is set as:

https://unpkg.com/moment@2.29.1/min/moment-with-locales.js

Not sure why it is not working?

**Comment by robbhar** [comment]
Oh on reboot is says moment.js not defined?

**Comment by tomasrudh** [comment]
It is most probably a caching issue. What you also can try is to download 'moment.js' and refer to that local file, instead of the URL.
